OBF Bylaws.md

```diff
@@ -41,7 +41,7 @@ ARTICLE II - MEETINGS
 ARTICLE III - BOARD OF DIRECTORS
 ----------------------------
 
-1.	The Board of Directors shall consist of President, Secretary, Treasurer, and additional Board Members at Large. The number of Board Members at Large is decided by majority vote of the current Board of Directors and may be proposed by either the general membership or a Board Member.
+1.	The Board of Directors shall consist of President, Deputy President, Treasurer, and additional Board Members at Large. The number of Board Members at Large is decided by majority vote of the current Board of Directors and may be proposed by either the general membership or a Board Member.
```

Comment:
Maybe I missed this decision, but why "Deputy President"? Isn't Vice President a more widely used term? (And a vice president similarly is empowered to act on behalf of the president if the president is not available.)

Reply:
+1 for vice president.

Reply:
I'll change it. The terms seem to be used interchangeably, perhaps with a UK/US bias. We have a Deputy Prime Minister, you have a Vice President ;)